test/controllers/supervisor_runs_controller_test.rb

```diff
@@ -1,36 +1,75 @@
 require 'test_helper'
 
 class SupervisorRunsControllerTest < ActionController::TestCase
+  ## workaround for MiniTest errors on some configurations
+  tests SupervisorRunsController
 
-  ID = 'id'
+  SUPERVISOR_ID = 'supervisor_id'
+  EXPERIMENT_ID = 'experiment_id'
+  STATE = '{state: :state}'
```

Comment:
Metoda SupervisorRun.state zwraca Hash - dlaczego tutaj jest String? Później jest assert_equal [STATE], JSON.parse(...) - dziwi mnie, że to działa, bo porównuje tabilcę, która powinna mieć w środku hasha z tabilcą która ma w środku stringa (?)